CONTRIBUTING.md

```diff
@@ -1,18 +1,121 @@
+
+* **Map style issues** should be reported in a [separate repository](https://github.com/ENT8R/streetcomplete-mapstyle).
+Examples of such issues are missing elements on the map, display errors on the map, etc. As a rule of thumb, you can report everything that happens "behind" the quest icon markers there.
+
+Note that this app has other dependencies. For reporting issues, in these, you however have to have some technical knowledge. So if you are **not sure** that the component listed below is responsible for the issue you have, it is often better to report them in the general StreetComplete issue tracker. People will then let you know whether this issue is solvable by StreetComplete or is an issue of Tangram-ES.
+The full list of dependencities and other StreetComplete-related projects is listed [at the end of this file](#streetcomplete-related-projects).
+
+### Suggesting new quests
+
+We appreciate all quest ideas you and others have, but some ideas cannot be added into the app due to various design decisions and requirements. This is required in order to make a great app for all users of StreetComplete, whose contributions actually help the OpenStreetMap project. A quest which nobody can solve and where users potentially choose wrong options adding wrong data to OpenStreetMap hurts not only OSM, but also this app.
+
+What you need to do:
+
+1. Search whether the quest you want to suggest is [already implemented in StreetComplete](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or [already suggested in the issue tracker](https://github.com/westnordost/StreetComplete/issues).
+2. Please read the full guide on what requirements you need to consider to make the quest a great thing: [Adding new Quests to StreetComplete](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete).
+3. Open a new issue and fill out the issue template. Explain why your quest meets the criteria you need to check, so other users can follow the example. Keep everything short (i.e. you do not have to write a wall of text there), but precise!
+4. Wait for input of others, even if you could implement it right away. It is always suggested to _first_ suggest a quest and implement it only if it has been accepted there.
+
+➡️ If you can code, see also how to [develop your own quest](#developing-new-quests).
+
+A quest is officially marked as accepted when it get's [the "new quest" tag](https://github.com/westnordost/StreetComplete/labels/new%20quest).
+<!-- TODO: introduce a better way/new tag here? ref https://github.com/westnordost/StreetComplete/issues/1006 -->
+
+## Improving documentation
+
+You can also help to keep the OpenStreetMap wiki **up-to-date** for StreetComplete. For example:
+* Add missing quests in [the quest list](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or check, that they are up to date.
+* Edit [the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) and add reoccuring questions.
```

Comment:
According to the [Oxford dictionary](https://en.oxforddictionaries.com/definition/reoccur) this should be reoccur**r**ing (with a second 'r')